frontend/package.json

```diff
@@ -32,8 +33,7 @@
     "semantic-ui-css": "^2.4.1",
     "semantic-ui-react": "^2.0.1",
     "styled-components": "^5.2.1",
-    "suneditor-react": "^2.15.2",
-    "web-vitals": "^0.2.4"
```

Comment:
why this package has been removed?

Reply:
> why this package has been removed?

Yes, i am installing it again. I didnt remove but somehow it got reoved. I am doing it